repository/src/main/java/org/apache/atlas/repository/patches/AtlasPatchManager.java

```diff
@@ -48,28 +48,24 @@ public AtlasPatchManager(AtlasGraph atlasGraph, AtlasTypeRegistry typeRegistry,
         this.context = new PatchContext(atlasGraph, typeRegistry, indexer, entityGraphMapper);
     }
 
-    @PostConstruct
```

Comment:
This approach is fine. However, i would keep the init method in tact and just remove the @PostConstruct annotation.

Call _init_ in the _applyAll_ method.

I verified this change and it is fine.